source/future.tex

```diff
@@ -1271,7 +1271,7 @@
 To enable old function adaptors to manipulate function objects
 that take one or two arguments,
 many of the function objects in this standard
-correspondingly provide typedefs
+correspondingly provide type aliases
```

Comment:
Maybe `\grammarterm{typedef-name}{s}`, to be clear that we don't care whether they're introduced by a `typedef` declaration or by an *alias-declaration*? "Type alias" is not a defined term in the standard.